packages/manager/apps/web/client/app/components/hosting-domain-offers/controller.js

```diff
@@ -124,7 +117,7 @@ export default class WebComponentsHostingDomainOffersController {
 
           return {
             ...offer,
-            badge: this.getOfferBadge(offer),
+            badge: this.constructor.getOfferBadge(),
```

Comment:
To be consistent with the current code to access use function. 
```suggestion
            badge: WebComponentsHostingDomainOffersController.getOfferBadge(),
```